waiter/resources/log4j.properties

```diff
@@ -38,3 +38,6 @@ log4j.appender.SchedulerAppender.File=log/${waiter.logFilePrefix}scheduler.log
 log4j.appender.SchedulerAppender.DatePattern='.'yyyy-MM-dd
 log4j.appender.SchedulerAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.SchedulerAppender.layout.ConversionPattern=%d{ISO8601} %-5p %c [%t] - [CID] %m%n
+
+log4j.logger.org.apache.http=DEBUG
+log4j.logger.org.eclipse.jetty=DEBUG
```

Comment:
We ned to restore this file before merging.

Reply:
Removed.